contracts/ColonyNetworkAuction.sol

```diff
@@ -25,11 +25,24 @@ contract ColonyNetworkAuction is ColonyNetworkStorage {
   event AuctionCreated(address auction, address token, uint256 quantity);
 
   function startTokenAuction(address _token) public stoppable {
+    require(_token != 0x0, "colony-auction-invalid-token");
+
     uint lastAuctionTimestamp = recentAuctions[_token];
     require(lastAuctionTimestamp == 0 || now - lastAuctionTimestamp >= 30 days, "colony-auction-start-too-soon");
+
     address clny = IColony(metaColony).getToken();
-    DutchAuction auction = new DutchAuction(clny, _token, metaColony);
+    require(clny != 0x0, "colony-auction-invalid-token");
+
     uint availableTokens = ERC20Extended(_token).balanceOf(this);
+
+    if (_token==clny) {
+      // We don't auction CLNY. We just burn it instead.
+      // Note we can do this more often than every 30 days.
+      ERC20Extended(clny).burn(availableTokens);
```

Comment:
ColonyNetwork will receive CLNY from:
- MetaColony task payout fees
- MetaColony rewards payout fees
(no longer auctions since this PR)

The above will be burned here so wanted to make sure we're okay with that.